docs/source/deployment/kubernetes/prerequisites.rst

```diff
@@ -0,0 +1,75 @@
+Corda Kubernetes Deployment option prerequisites
+================================================
+
+.. line-block::
+    The Corda Kubernetes Deployment option has the following prerequisites:
+
+    * A cloud environment with Kubernetes Cluster Services that has access to a Docker Container Registry
+        (Note! The current version of the scripts only supports Azure out of the box by way of Azure Kubernetes Service and Azure Container Registry, future versions of the scripts may add support for other cloud providers)
+    * Building the images requires local Docker installation `Docker <https://www.docker.com/>`_
+    * kubectl is used to manage Kubernetes cluster `Install kubectl <https://kubernetes.io/docs/tasks/tools/install-kubectl/>`_
+    * `Helm <https://helm.sh/>`_ 
+        Helm allows us to store the Kubernetes deployment scripts without usecase specific information in Git.
+        Helm will then have a values.yaml file that defines all the customizable fields and settings to use when deploying to Kubernetes.
+    * Corda Enterprise jars downloaded and stored in 'bin' folder
+        Downloading these requires a valid Corda Enterprise License agreement.
+        If you have the necessary login details you can download them directly from: `Corda Artifactory <https://ci-artifactory.corda.r3cev.com/artifactory/webapp/#/home>`_
+       
+Setup
+~~~~~
+
```

Comment:
Suggestion : I'd add a disclaimer here that support for the setup of any Azure services should go to Microsoft, not R3. R3 Solution Engineering does not support  users with set up of Azure deployments.